forms-flow-web/src/components/Form/constants/FormTable.js

```diff
@@ -109,7 +119,50 @@ function FormTable() {
       </tbody>
     );
   };
-
+  const SortableHeader = ({ columnKey, title, formSort, handleSort,className }) => {
+    const handleKeyDown = (event)=>{
+      if (event.key === 'Enter') {  
+        handleSort(columnKey);
+        }
+    };
+    return (
+      <div
+        className= {`d-flex align-items-center justify-content-between cursor-pointer ${className}`}
+        onClick={() => handleSort(columnKey)}
+        onKeyDown={handleKeyDown} 
+        role="button"
+      >
+        <span className="mt-1">{t(title)}</span>
+        <span>
+          {formSort.sortBy === columnKey && formSort.sortOrder === "asc" ? (
+            <i
+              data-testid={`${columnKey}-desc-sort-icon`}
+              className="fa fa-arrow-up sort-icon fs-16 ms-2"
+              data-toggle="tooltip"
```

Comment:
Is this SortableHeader a reusable item (which we can use on other tables like DMN, BPMNs) , then can we convert to reusable file?
 @fahad-aot

Reply:
we can convert  that into reusable file after checking BPMN or DMN table . will add in that PR .